manifest.yaml

```diff
@@ -0,0 +1,36 @@
+applications:
+- name: apiserver
+  memory: 512M
+  instances: 1
+  host: apiserver
+  path: api/
+- name: servicebroker
+  memory: 512M
+  instances: 1
+  host: servicebroker
+  path: servicebroker
+- name: eventgenerator
```

Comment:
Go app files are directly pushed. For automatic buildpack detection we should add GOPACKAGENAME environment variable also for dependencies we need some vendoring(maybe instructions can be added in the readme).
Same for metricscollector, scalingengine and pruner too.